_posts/2024-11-14-robotics1.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: Robotics
+tweetText: Robotics
```

Comment:
hey name?

Reply:
This git page says:

https://git-scm.com/downloads/win